app/experimenter/visualization/tests/api/test_views.py

```diff
@@ -16,6 +16,8 @@
 
 @override_settings(FEATURE_ANALYSIS=False)
 class TestVisualizationView(TestCase):
+    maxDiff = None
```

Comment:
Do you wanna keep this?

Reply:
Yeah those are fine to leave in that way if it ever fails again we'll get the full dump